From 219f80b5cc03dab87fd05210b95c0b1a5afa8d33 Mon Sep 17 00:00:00 2001
From: Russell King <rmk+kernel@armlinux.org.uk>
Date: Thu, 14 Jul 2016 15:31:42 +0100
Subject: ARM: dts: armada388-clearfog: use 1000BaseX mode for 88e6176 switch

Use 1000BaseX mode for the 88e6176 switch, which allows mvneta to
negotiate correctly without needing to be forced.

Signed-off-by: Russell King <rmk+kernel@armlinux.org.uk>
---
 arch/arm/boot/dts/armada-388-clearfog.dts | 10 ++--------
 1 file changed, 2 insertions(+), 8 deletions(-)

--- a/arch/arm/boot/dts/armada-388-clearfog.dts
+++ b/arch/arm/boot/dts/armada-388-clearfog.dts
@@ -47,10 +47,8 @@
 
 &eth1 {
 	/* ethernet@30000 */
-	fixed-link {
-		speed = <1000>;
-		full-duplex;
-	};
+	phy-mode = "1000base-x";
+	managed = "in-band-status";
 };
 
 &expander0 {
@@ -131,10 +129,6 @@
 				reg = <5>;
 				label = "cpu";
 				ethernet = <&eth1>;
-				fixed-link {
-					speed = <1000>;
-					full-duplex;
-				};
 			};
 
 			port@6 {
